rpcd/playbooks/roles/rpc_maas/templates/galera_check.yaml.j2

```diff
@@ -24,3 +23,14 @@ alarms      :
             if (metric["wsrep_local_state_comment"] != "Synced" ) {
                 return new AlarmStatus(CRITICAL, "Galera cluster node not synced");
             }
+    percentage_used_mysql_connections :
+        label                   : percentage_used_mysql_connections--{{ ansible_hostname }}
+        notification_plan_id    : "{{ maas_notification_plan }}"
+        criteria                : |
+            :set consecutiveCount={{ maas_alarm_local_consecutive_count }}
+            if (percentage(metric["mysql_current_connections"], metric["mysql_max_configured_connections"]) > {{ mysql_connection_warning_threshold }} ) {
+                return new AlarmStatus(WARNING, "Mysql connectons has exceeded {{ mysql_connection_warning_threshold }}% of configured maximum");
```

Comment:
warning and critical conditions should be swapped otherwise the alarm will never go critical.